src/web/jstl/spec/fmt/format/localecontext/positiveFormatLocalizationContextBrowserLocaleTestJava20Plus.jsp

```diff
@@ -0,0 +1,41 @@
+<%--
+
+    Copyright (c) 2003, 2021 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Same as above since this is a new file just need `Copyright (c) 2024 Contributors to the Eclipse Foundation`.